R/ppc-intervals.R

```diff
@@ -173,14 +173,17 @@ ppc_intervals_grouped <- function(y,
 
 #' @rdname PPC-intervals
 #' @export
+#' @param y_draw For ribbon plots only, a string specifying how to draw `y. Can
+#'   be `"line"` (the default), `"points"`, or `"both"`.
 ppc_ribbon <- function(y,
                        yrep,
                        x = NULL,
                        ...,
                        prob = 0.5,
                        prob_outer = 0.9,
                        alpha = 0.33,
-                       size = 0.25) {
+                       size = 0.25,
+                       y_draw = c("line", "points", "both")) {
```

Comment:
Not essential, but it's common in R to put all valid options in a vector and then use `match.arg` later to ensure that what the use wrote is a valid option. (See below where I added `match.arg`)

Reply:
Makes sense. I was wondering what match.arg does.